README.adoc

```diff
@@ -416,8 +415,22 @@ kubectl create secret generic sys-app-credentials \
         --from-literal password=[password]
 ----
 
+To learn more how to manage the secrets, visit the https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl[Managing Secrets using kubectl^] documentation.
+
+
 Finally, write up the [hotspot file=0]`deploy.yaml` deployment file to configure the deployment of the [hotspot=system file=0]`system` and [hotspot=query file=0]`query` microservices by using the Open Liberty Operator. The [hotspot=sys-app-credentials-1 hotspot=sys-app-credentials-2 hotspot=sys-app-credentials-3 hotspot=sys-app-credentials-4 file=0]`sys-app-credentials` Kubernetes secrets set the environment variables [hotspot=default.username file=0]`DEFAULT_USERNAME` and [hotspot=default.password file=0]`DEFAULT_PASSWORD` for the `system` microservice, and [hotspot=system.user file=0]`SYSTEM_USER` and [hotspot=system.password file=0]`SYSTEM_PASSWORD` for the `query` microservice.
 
+// file 5
+deploy.yaml
+[source, XML, linenums, role='code_column']
+----
+include::finish/deploy.yaml[]
+----
+
+
+If you want to override another property, you can specify it through the `env` sections in the `deploy.yaml`. For example, set the [hotspot=context.root file=5]`CONTEXT_ROOT` environment variable in the `system` deployment and the [hotspot=system.contextRoot file=5]`SYSTEM_CONTEXTROOT` environment variable in the `query` deployment.
```

Comment:
> set the `CONTEXT_ROOT` environment variable in the `system` deployment and the `SYSTEM_CONTEXTROOT` environment variable in the `query` deployment.

Is this right? seems confusing that the SYSTEM_CONTEXTROOT variable goes with the `query` instead of the `system`

Reply:
It is right. In the system service perspective, CONTEXT_ROOT is for its own context root.  In the query service perspective, it needs to know the context root of the system service, so SYSTEM_CONTEXTROOT.